SQL/0000-00-02-Menus.sql

```diff
@@ -15,7 +15,8 @@ INSERT INTO LorisMenu (Label, OrderNumber) VALUES
      ('Imaging', 3), 
      ('Reports', 4), 
      ('Tools', 5), 
-     ('Admin', 6);
+     ('Admin', 6),
+     ('Acknowledgements', 7);;
```

Comment:
It would probably make more sense to put this under "Tools" instead of another top level menu with 1 item. (Also, there's 2 semicolons here.